src/VS.Web.CG.Mvc/Common/RequiredFileEntity.cs

```diff
@@ -24,13 +25,10 @@ public RequiredFileEntity(string outputPath, string templateName)
             OutputPath = outputPath;
         }
         
-        public RequiredFileEntity(string outputPath, string templateName, List<string> altPaths)
+        public RequiredFileEntity(string outputPath, string templateName, IEnumerable<string> altPaths)
             : this(outputPath, templateName)
         {
-            if (altPaths != null)
-            {
-                AltPaths = altPaths;
-            }
+            AltPaths = altPaths.ToList() ?? new List<string>();
```

Comment:
This may result in a NRE.
Use `altPaths?.ToList()` to avoid it.

Reply:
Also, you don't need the `= new List<string>()` any more for the `AltPath` property